records/2022-04-14.secrets-usage.md

```diff
@@ -0,0 +1,123 @@
+# Secrets in Notify
+
+Date: 2022-04-14
+
+## Status
+
+**DRAFT**.
+
+## Context
+
+With recent secrets leaked to the public, we have to familiarize ourselves with how the secrets
+we have defined within Notify are used. This document aims to explain exactly how these interact
+with different components of our application.
+
+## Methods and libraries
+
+This section enumerates a few methods and libraries to help understand the rest of
+the document to casual readers.
+
+### [itsdangerous](https://itsdangerous.palletsprojects.com/)
+
+This is a library that specializes on signing content in a cryptographic manner.
+A piece of content that is signed with a [HMAC-SHA1 algorithm](https://en.wikipedia.org/wiki/HMAC)
+algorithm. This will not encrypt a message in a one-way manner; we can reverse
```

Comment:
```suggestion
This is a library that specializes in signing content in a cryptographic manner.
using a [HMAC-SHA1 algorithm](https://en.wikipedia.org/wiki/HMAC). This will not encrypt a message in a one-way manner; we can reverse
```